### DISCUSSION TITLE
Welcome to ExtremeCodeOS Discussions!

### DISCUSSION THREAD

**Discussion by extremecodetv** [extremecodetv/ExtremeCodeOS#100] [Announcements]
раз, два, раз, это че за фичя новая?

**Comment by hawr-desk** [comment]
> раз, два, раз, это че за фичя новая?

Ей вроде бы год. Дискуссии по коду (обычно).
Иногда используется как StackOverFlow

**Reply by extremecodetv** [reply]
хз, у меня сегодня только появилось на всех репозиториях

**Comment by hawr-desk** [comment]
Кстати, я давно хотел спросить. Зачем нужен файл kernel.c? Если мы делаем новый дистрибутив, а не новое ядро.

**Reply by ghost** [reply]
Чтобы поугарать
